eventstore/sql/create_events.sql

```diff
@@ -310,8 +310,17 @@ INSERT INTO events with
 		event_sequence, event_guid
 ;
 
-CREATE INDEX events_org_idx ON events (org_guid);
-CREATE INDEX events_space_idx ON events (space_guid);
-CREATE INDEX events_resource_idx ON events (resource_guid);
-CREATE INDEX events_duration_idx ON events using gist (duration);
-CREATE INDEX events_plan_idx ON events (plan_guid);
+
+CREATE INDEX events_org_temp_idx ON events_temp (org_guid);
+CREATE INDEX events_space_temp_idx ON events_temp (space_guid);
+CREATE INDEX events_resource_temp_idx ON events_temp (resource_guid);
+CREATE INDEX events_duration_temp_idx ON events_temp using gist (duration);
+CREATE INDEX events_plan_temp_idx ON events_temp (plan_guid);
+DROP TABLE IF EXISTS events;
```

Comment:
Might help to have a bit of extra whitespace in here.

```suggestion

DROP TABLE IF EXISTS events;

```